test_2_uppercases_all_letters.py

```diff
@@ -1,6 +1,5 @@
 def uppercases_all_letters(str):
-  # Remove the `return None` code below and add your own code.
-  return None
+  return str.upper
```

Comment:
I just tested this in C9 successfully...I think it was my upper function not having parentheses on the end. Resubmitted!